TerriaJS/terriajs
# Allow user to specify colours for csv data in WPS responses

If a json/csv ComplexData contains 'color' as an attribute (which is a comma separated list of css colors), these colours are applied to the columns of the csv. It allows the user to override the default behaviour of picking a colour that hasn't been used from a list of possible colours.

To test: 
1. With this change: https://github.com/TerriaJS/nationalmap/pull/373
2. In #test, Test Data > WPS > Csv Output (if you don't see that, you don't have the change in step 1)
3. Select a point on the map
4. Run analysis
5. Note that the colours for the chart are a strong red, green, blue, whereas previously they'd be orange, light green, purple.
